supply/lib/supply/options.rb

```diff
@@ -121,7 +121,13 @@ def self.available_options
                                      verify_block: proc do |value|
                                        available = valid_tracks
                                        UI.user_error! "Invalid value '#{value}', must be #{available.join(', ')}" unless available.include? value
-                                     end)
+                                     end),
+        FastlaneCore::ConfigItem.new(key: :validate_only,
+                                     env_name: "SUPPLY_VALIDATE_ONLY",
+                                     optional: true,
+                                     description: "If true, changes will only be validated with Google Play rather than actually uploaded",
```

Comment:
`Indicate that changes will only be validated with Google Play rather than actually published` would sound better. Also I think that your data is still uploaded - just the publishing step is skipped?